### DISCUSSION TITLE
Why is no geometry specified for Getting Started example "4.1 Create a new egs++ application"?

### DISCUSSION THREAD

**Discussion by jburz2001** [nrc-cnrc/EGSnrc#1135] [Questions] [answered]
Hello. I'm brand new to EGSnrc and am trying to go through the Getting Started PDF. I think I have every required dependency installed correctly so I tried running the first example in section _4.1 Create a new egs++ application_ (https://nrc-cnrc.github.io/EGSnrc/doc/getting-started.pdf) but am getting some errors. In particular, I get the following as output when running the executable generated by running `make` in the myapp directory which has the `Makefile` in it: 
![image](https://github.com/nrc-cnrc/EGSnrc/assets/56214083/bd905961-36b3-433d-a56c-f893edbb157f).

Why might this be? After I run make in the directory at path `~/EGSnrc/egs_home/myapp/` the following files are created: 
![image](https://github.com/nrc-cnrc/EGSnrc/assets/56214083/08f267c3-0da2-41e2-b4ad-dc410c4300ee).

The files I used with `make` are included in the attached .zip file. The myapp.macros is empty:
[4pt1_files.zip](https://github.com/nrc-cnrc/EGSnrc/files/15485214/4pt1_files.zip)

**Comment by rtownson** [comment, marked as answer]
Your compilation with `make` was successful! Next you just have to do section 4.2 of the getting started manual, to create an input file. After you have created the input file (e.g. `slab.egsinp`), then you can run the application using
```
myapp -i slab.egsinp
```
Almost all EGSnrc applications require an input file to be provided as an argument when you run the application.

**Reply by jburz2001** [reply]
Thank you very much for letting me know. I ran 4.2 and it worked! 
![image](https://github.com/nrc-cnrc/EGSnrc/assets/56214083/ab717964-b3b2-469f-81b7-32b73d5d534c)

**Reply by jburz2001** [reply]
I took a look at the output in the terminal and it says `QObject::connect: No such signal QLineEdit::lostFocus()`. Is this an issue?

![image](https://github.com/nrc-cnrc/EGSnrc/assets/56214083/14bdf3bc-3068-4516-a9f5-07627c75de39)

**Reply by rtownson** [reply]
This is normal, nothing to worry about.
